atr/get/distribution.py

```diff
@@ -66,55 +68,116 @@ async def list_get(session: web.Committer, project: str, version: str) -> str:
     ## Distributions
     block.h2["Distributions"]
     for dist in distributions:
-        delete_form = await shared.distribution.DeleteForm.create_form(
-            data={
-                "release_name": dist.release_name,
-                "platform": dist.platform.name,
-                "owner_namespace": dist.owner_namespace,
-                "package": dist.package,
-                "version": dist.version,
-            }
-        )
-
         ### Platform package version
         block.h3(
             # Cannot use "#id" here, because the ID contains "."
             # If an ID contains ".", htm parses that as a class
             id=f"distribution-{dist.identifier}"
         )[dist.title]
         tbody = htm.tbody[
-            shared.distribution.html_tr("Release name", dist.release_name),
-            shared.distribution.html_tr("Platform", dist.platform.value.name),
-            shared.distribution.html_tr("Owner or Namespace", dist.owner_namespace or "-"),
-            shared.distribution.html_tr("Package", dist.package),
-            shared.distribution.html_tr("Version", dist.version),
-            shared.distribution.html_tr("Staging", "Yes" if dist.staging else "No"),
-            shared.distribution.html_tr("Upload date", str(dist.upload_date)),
-            shared.distribution.html_tr_a("API URL", dist.api_url),
-            shared.distribution.html_tr_a("Web URL", dist.web_url),
+            shared.html_tr("Release name", dist.release_name),
+            shared.html_tr("Platform", dist.platform.value.name),
+            shared.html_tr("Owner or Namespace", dist.owner_namespace or "-"),
+            shared.html_tr("Package", dist.package),
+            shared.html_tr("Version", dist.version),
+            shared.html_tr("Staging", "Yes" if dist.staging else "No"),
+            shared.html_tr("Upload date", str(dist.upload_date)),
+            shared.html_tr_a("API URL", dist.api_url),
+            shared.html_tr_a("Web URL", dist.web_url),
         ]
         block.table(".table.table-striped.table-bordered")[tbody]
-        form_action = util.as_url(post.distribution.delete, project=project, version=version)
-        delete_form_element = forms.render_simple(
-            delete_form,
-            action=form_action,
-            submit_classes="btn-danger",
-        )
-        block.append(htm.div(".mb-3")[delete_form_element])
+
+        # Create inline delete form with confirmation dialog (following projects.py pattern)
```

Comment:
This is currently the correct way to do this, but please add a TODO to the effect that we should add a `confirm: bool` keyword argument to `form.render`. This will allow us to use `form.render` and not have to special case forms with confirmations.